spack/g/gcc/package.py

```diff
@@ -28,43 +28,355 @@ class Gcc(AutotoolsPackage, GNUMirrorPackage):
     maintainers = ['michaelkuhn', 'alalazo']
 
     version('master', branch='master')
+    version('11.2.0', sha256='d08edc536b54c372a1010ff6619dd274c0f1603aa49212ba20f7aa2cda36fa8b')
```

Comment:
@alecbcs this looks like that redundancy bug again!

Reply:
Yep! Hopefully this should now be fixed. Closing this and we'll see if it happens again.